<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo49</class>
 <widget class="QDialog" name="dlgDemo49">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtrar Por Cualquier Columna usando QSortFilterProxyModel</string>
  </property>
  <widget class="QTableView" name="tblProducto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>761</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalMensaje">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total de Productos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalRegistros">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>500</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblValor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor a Filtrar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtValor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFiltrar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCampoFiltrar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Campo a Filtrar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboCampo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboOperador">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblOperadorFiltro">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operador del Filtro</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
